dali/test/python/operator_2/test_remap.py

```diff
@@ -145,8 +145,6 @@ def benchmark_remap_against_cv(self, map_mode):
             **self.common_dali_pipe_params,
             prefetch_queue_depth=1,
         )
-        dpipe.build()
-        cpipe.build()
```

Comment:
This is a benchmark; please restore, we don't want to measure `build`.

Reply:
reverted